lib/ios/RNNOverlayManager.m

```diff
@@ -29,10 +29,11 @@ - (void)dismissOverlay:(UIViewController *)viewController {
     [self detachOverlayWindow:overlayWindow];
 }
 
-- (void)dismissAllOverlays {
+- (void)dismissAllOverlays:(void (^__nullable)(void))completion {
```

Comment:
Why is a completion block needed? The method will block the thread anyway until the overlays are detached, so there is advantage on using a completion block.

Reply:
Because we need to wait for all of the overlays to be dismissed then we'll call completion() in the dismissAllOverlay method that will let JS land to know that everything is done.

Reply:
Yes, but the method `dismissAllOverlays` is synchronous anyway.

Reply:
Oh, you are right. I'll change it.